auth/store.go

```diff
@@ -207,17 +218,21 @@ func (as *authStore) AuthDisable() {
 	tx.Unlock()
 	b.ForceCommit()
 
-	as.enabledMu.Lock()
 	as.enabled = false
-	as.enabledMu.Unlock()
 
 	as.simpleTokensMu.Lock()
 	as.simpleTokens = make(map[string]string) // invalidate all tokens
+	as.simpleTokenKeeper.stop()
```

Comment:
this is already protected by simpleTokensMu.Lock(), why hold as.enabledMu for a longer time?